src/Config/Settings.ts

```diff
@@ -102,7 +102,7 @@ const defaultSettings: Settings = {
     setCancelledDate: true,
     autoSuggestInEditor: true,
     autoSuggestMinMatch: 0,
-    autoSuggestMaxItems: 6,
+    autoSuggestMaxItems: 20,
```

Comment:
20 seems a lot to me. What was the thinking behind this number?
What does 20 look like on a mobile phone screen?
Does Obsidian even show 20 suggestions?
And does it scroll the list if there are more suggestions than fit on the screen?

Reply:
20 is actually really quite useful when doing dependency searches, so I will leave it as it is.

Reply:
Resolving conversation.